test/iostream.jl

```diff
@@ -0,0 +1,19 @@
+@test mktemp() do _, file
+    skipchars(file, isspace)
+    true
+end
+
+@test eof(skipchars(IOBuffer("    "), isspace))
+@test eof(skipchars(IOBuffer("#    \n   "), isspace, linecomment='#'))
+
+macro test_skipchars(str, expected_char, lnc=Char(0xffffffff))
+  quote
```

Comment:
[[ Had an appointment ]]
Done, thanks for catching these! I will squash it once all the work is finished.